### ISSUE TITLE
Cmake issues an error when making SIPI from scratch

### ISSUE THREAD

**Issue by lrosenth** [dhlab-basel/Sipi#208]
```
-- Found Git: /usr/bin/git (found version "2.11.0") 
-- Git version: 67a377c
-- Configuring done
CMake Error at test/unit/sipiimage/CMakeLists.txt:23 (add_executable):
  Cannot find source file:

    /sipi/include/AdobeRGB1998_icc.h

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx


-- Generating done
-- Build files have been written to: /sipi/build
The command '/bin/sh -c cd /sipi/build &&     cmake .. 	&& make 	&& cp sipi /usr/local/bin 	&& rm -rf /sipi/vendor     && rm -rf /sipi/build     && rm -rf /sipi/extsrcs' returned a non-zero code: 1
```

This prevents clean Dockerfile building

@subotic

**Comment by subotic** [comment]
There seems to be no `AdobeRGB1998_icc.h` file anymore. In that case, you can simply remove it from the `test/unit/sipiimage/CMakeLists.txt`.

**Comment by subotic** [comment]
I'm trying it out right now and will create a PR.

**Comment by subotic** [comment]
hmm, on Travis, there is no such error, and `sipi` is build inside docker with this command: `docker run -v $PWD:/sipi dhlabbasel/sipi-base /bin/sh -c "cd /sipi/build; cmake ..; make; make test; make check"`

**Comment by subotic** [comment]
I have run a clean `sipi` native mac build and through docker, and had no issues.

**Comment by subotic** [comment]
fixed in #210
